conjure-java-client-verifier/src/test/resources/ignored-test-cases.yml

```diff
@@ -66,6 +66,10 @@ client:
    receiveSetAnyAliasExample:
    - 'null'
 
+   receiveEnumExample:
+   - '"!!!"'
+   - '"one-hundred"'
```

Comment:
We're intentionally not failing on these as we decided to make conjure-java intentionally more lenient than the spec. Could we put a comment saying something to this effect? (e.g. possibly linking to https://github.com/palantir/conjure-java/pull/238)